indexers/vector/PysparnnIndexer/tests/test_pysparnnindexer.py

```diff
@@ -43,6 +43,12 @@ def test_add(indexer, features):
     assert isinstance(indexer.index[0], csr_matrix)
 
 
+def test_add_newkeys(indexer, features):
```

Comment:
This does not test adding new keys,every time the `indexer` fixture is created, a new indexer is created with no data.

Reply:
Done